Ritual for checking new versions of macOS and CIS benchmarks (macOS and Windows Enterprise) (#12431)
fleetdm/fleet: handbook/product/README.md

In order to understand the usage of the Fleet product, we [collect statistics](https://fleetdm.com/docs/using-fleet/usage-statistics) from installations where this functionality is enabled.

Fleeties can view these statistics in the Google spreadsheet [Fleet
usage](https://docs.google.com/spreadsheets/d/1Mh7Vf4kJL8b5TWlHxcX7mYwaakZMg_ZGNLY3kl1VI-c/edit#gid=0)
available in Google Drive.

> Some of the data is forwarded to a [DataDog board](https://us5.datadoghq.com/dashboard/7pb-63g-xty/usage-statistics?from_ts=1682952132131&to_ts=1685630532131&live=true) and is available to Fleeties.

Reviewer: Datadog

Author: ```suggestion
Some of the data is forwarded to [DataDog](https://us5.datadoghq.com/dashboard/7pb-63g-xty/usage-statistics?from_ts=1682952132131&to_ts=1685630532131&live=true) and is available to Fleeties.
```

## Maintenance

> Fleet's product offerings depend on the capabilities of other platforms. This require the ongoing attention of the product and engineering teams to ensure that we are up-to-date with new capabilities and our existing capabilities continue to function. The first step to staying up-to-date with our partners is to know when the partner platform changes. 

Reviewer: This requires

Reviewer: and our => and that our

Reviewer: our partners => Fleet's partners

Author: ```suggestion
Fleet's product offerings depend on the capabilities of other platforms. This requires the ongoing attention of the product and engineering teams to ensure that we are up-to-date with new capabilities and that our existing capabilities continue to function. The first step to staying up-to-date with Fleet's partners is to know when the partner platform changes. 
```

> Every week, a member of the product team (as determined in the [rituals](#rituals) section) should look up whether there is:

Reviewer: should look => looks

Author: ```suggestion
Every week, a member of the product team (as determined in the [rituals](#rituals) section) looks up whether there is:
```

> 1. a new major or minor version of macOS (https://support.apple.com/en-us/HT201260)

Author: ```suggestion
1. a new major or minor version of [macOS](https://support.apple.com/en-us/HT201260)
```

> 1. a new major or minor version of MacOS (https://support.apple.com/en-us/HT201260)

Reviewer: macOS

Author: ```suggestion
1. a new major or minor version of macOS (https://support.apple.com/en-us/HT201260)
```

> 2. a new major or minor version of CIS Benchmarks Windows 10 Enterprise (https://workbench.cisecurity.org/community/2/benchmarks?q=windows+10+enterprise&status=&sortBy=version&type=desc)

Author: ```suggestion
2. a new major or minor version of [CIS Benchmarks Windows 10 Enterprise](https://workbench.cisecurity.org/community/2/benchmarks?q=windows+10+enterprise&status=&sortBy=version&type=desc)
```

3. a new major or minor version of CIS Benchmarks macOS 13 Ventura (https://workbench.cisecurity.org/community/20/benchmarks?q=macos+13.0+Ventura&status=&sortBy=version&type=desc)

> 4. a release of CIS Benchmarks for macOS 14 Sonoma (https://workbench.cisecurity.org/community/20/benchmarks?q=sonoma&status=&sortBy=version&type=desc)

Author: ```suggestion
4. a release of CIS Benchmarks for [macOS 14 Sonoma](https://workbench.cisecurity.org/community/20/benchmarks?q=sonoma&status=&sortBy=version&type=desc)
```

> 4. a release of CIS Benchmarks for MacOS 14 Sonoma (https://workbench.cisecurity.org/community/20/benchmarks?q=sonoma&status=&sortBy=version&type=desc)

Reviewer: macOS

Author: ```suggestion
4. a release of CIS Benchmarks for macOS 14 Sonoma (https://workbench.cisecurity.org/community/20/benchmarks?q=sonoma&status=&sortBy=version&type=desc)
```

> 5. a new major or minor version of ChromeOS (https://chromereleases.googleblog.com/search/label/Chrome%20OS)

Author: ```suggestion
5. a new major or minor version of [ChromeOS](https://chromereleases.googleblog.com/search/label/Chrome%20OS)
```

> 

Reviewer: light up relevant text above rather than including full url links

> If there are changes, the DRI should send a message in the #help-product slack channel, noting the current versions and whether any of the above has changed.

Reviewer: Slack

Author: ```suggestion
If there are changes, the DRI should send a message in the #help-product Slack channel, noting the current versions and whether any of the above has changed.
```

| ⚗️✨🗣 Design review (MDM)  | Daily | Review designs from the MDM team | Marko Lisica |
| ⚗️✨🗣 Design review (CX)   | Daily | Review designs from the CX team | Rachael Shaw |
| Product development process review | Sprintly | CEO, Director of Product Development, and Head of Product get together to review boards and process to make sure everything still makes sense | Mo Zhu |

> | New versions check | Weekly (Friday) | The DRI for this ritual checks conducts the maintenance checks as described in the [maintenance](#maintenance) section | Mo Zhu |

Reviewer: checks conducts

Author: ```suggestion
| New versions check | Weekly (Friday) | The DRI for this ritual conducts the maintenance checks as described in the [maintenance](#maintenance) section | Mo Zhu |
```
